CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ### Unreleased
 - Bug fixes:
 	- Removed "last user" cache to avoid overwriting anonymous user cache during init [SDKCF-6409]
+    - Added display permission service to perform ping on tooltip dispatcher [SDKCF-4964]
```

Comment:
@ts-robih-robihamanto Can you fix this indentation issue?

Reply:
Fixed here [6900708](https://github.com/rakutentech/ios-inappmessaging/pull/252/commits/8dbc1c83dcf1f84164118aa8ab4094f716900708).